pkg/rehearse/jobs.go

```diff
@@ -111,8 +111,14 @@ func makeRehearsalPresubmit(source *prowconfig.Presubmit, repo string, prNumber
 	rehearsal.Context = fmt.Sprintf("ci/rehearse/%s/%s/%s", repo, branch, shortName)
 	rehearsal.RerunCommand = defaultRehearsalRerunCommand
 
-	gitrefArg := fmt.Sprintf("--git-ref=%s@%s", repo, branch)
-	rehearsal.Spec.Containers[0].Args = append(source.Spec.Containers[0].Args, gitrefArg)
+	orgRepo := strings.Split(repo, "/")
+	rehearsal.ExtraRefs = append(rehearsal.ExtraRefs, pjapi.Refs{
+		Org:     orgRepo[0],
+		Repo:    orgRepo[1],
+		BaseRef: branch,
+		WorkDir: true,
```

Comment:
We might want to make sure that the other ones don't set this so we have only one workdir setting (does prow validate this?)

Reply:
`Prow` will use the first `workDir` that will find in the slice. `ci-operator` too... We don't validate multiple `workDir`s .. I guess this is good, if our stakeholders for some reason would want to override it

Reply:
Seems reasonable